content/config/otel.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Open Telemetry for the Docker CLI
+description: Learn about how you can leverage the Docker's capabilities to capture OTel traces
+keywords: otel, telemetry, traces, metrics, logs
+---
+
+{{< introduced engine 26.1.0 >}}
+
+The Docker CLI supports [OTel](https://opentelemetry.io/docs/) instrumentation
+for emitting metrics about command invocations. This is disabled by default.
+You can configure the CLI to start emitting metrics to the endpoint that you
+specify. This allows you to capture information about your `docker` command
+invocations for more insight into your Docker usage.
+
+The destination of these metrics is completely opt-in, and you control the
+destination to where the data is being sent.
+
+## What is OTel?
+
+OTel stands for Open Telemetry. OTel is an open observability framework for
+creating and managing telemetry data, such as traces, metrics, and logs.
+OpenTelemetry is vendor- and tool-agnostic, meaning that it can be used with a
```

Comment:
nit: extra - after vendor?

Reply:
I believe that's the correct way to indicate that it's a shorthand for `vendor-agnostic` and `tool-agnostic`, but I guess it could be maybe be rephrased as `vendor and tool agnostic`? Not sure which is clearer.

Reply:
I would prefer to keep the current format.